src/Logary/DataModel.fs

```diff
@@ -209,6 +209,29 @@ type Message =
     member x.timestampEpochS: int64 =
       x.timestamp / Constants.NanosPerSecond
 
+/// span focus on time scope (scope means it needs explicitly specify its end) which usually used for tracing.
+/// message focus on data context which usually used for logging/metric (since metric can be express by gauge which inclued by message).
+/// one message can belong to a span, it means this log message happened in this span.
+/// span itself contains one message, when logging, it just fill message with some specific info (startTime, endTime, duration, id ...)，
+/// then every target can transform this message into their own backend's implementation (like zipkin,jaeger,opentracing...).
+/// since span can be finished, so they will have a logger and send it's message at that time.
```

Comment:
Good description. I agree that this is a nice abstraction, but since Finish is async, should disposable also be optionally async (IAsyncDisposable)? I think it should be sync disposable, but then `finish` must be called without a caller (on the Hopac scheduler with a timeout, like `logSimple` does it)

Reply:
done. finish now is sync, use `logWith`(same way as logSimple).